Assets/Scripts/Camera/CameraFollow.cs

```diff
@@ -5,22 +5,43 @@
 public class CameraFollow : MonoBehaviour {
 
 	public Transform player;
-	public Vector3 offset;
+	public Vector3 	offset;
+	public float	ScreenshakeFrequency = 10f;
+
+	private float 	durationLeft = 0f,
+					intensity = 0f,
+					screenshakeC = 0f;
 
 	// Update is called once per frame
-	void Update()
+	void FixedUpdate()
 	{
-		transform.position = (player.position + offset);
+		Vector3 offsetExtra = Vector3.zero;
+		screenshakeC += Time.deltaTime * ScreenshakeFrequency;
 
-		if (Input.GetKeyDown ("q")) {
-		
-		}
-		if (Input.GetKeyDown ("e")) {
-		
+		// Screenshake
+		if (durationLeft > 0f)
+		{
+			durationLeft -= Time.deltaTime;
+			if ( screenshakeC >= 1f) 
+			{
+				screenshakeC = 0f;
+				float intensity0 = intensity;
+				if (durationLeft <= 0.1f) intensity0 *= durationLeft / 0.1f;
+				offsetExtra += new Vector3(Random.Range(-intensity0, intensity0), Random.Range(-intensity0, intensity0), Random.Range(-intensity0, intensity0));
+			}
 		}
+		transform.position = (player.position + offset + offsetExtra);
 	}
 
-	void rotateCamera(int dir){
-		
+	/**
```

Comment:
Nice that you include function comments, but this is not the format that we have used in other parts of the project